ocsci/tests/test_main.py

```diff
@@ -15,31 +26,25 @@ def test_help(self):
         assert '--cluster-path' in result
         assert '--cluster-name' in result
 
+    @mock.patch('ocsci.main.pytest.main')
+    @mock.patch.object(config, 'update')
+    def test_no_args(self, config_update, pytest_main, testdir):
+        main.main([])
+        assert config_update.call_count == 0
 
-class TestUpdateDict(object):
-    def test_update_dict(self):
-        objA = dict(
-            a_dict=dict(
-                another_list=[],
-                another_string="greetings",
-            ),
-            a_list=[0, 1, 2],
-        )
-        objB = dict(
-            a_dict=dict(
-                a_third_string="salutations",
-            ),
-            a_string='string',
-        )
-        expected = dict(
-            a_dict=dict(
-                another_list=[],
-                another_string="greetings",
-                a_third_string="salutations",
-            ),
-            a_list=[0, 1, 2],
-            a_string='string',
-        )
-        result = update_dict_recursively(objA, objB)
-        assert objA is result
-        assert result == expected
+    @mock.patch('ocsci.main.pytest.main')
+    @mock.patch.object(config, 'update')
+    def test_config_passing(self, config_update, pytest_main, testdir):
+        tempdir = testdir.makefile(
+            '.yaml',
+            ocsci_conf='RUN: null',
+            cluster_conf='DEPLOYMENT: null',
+        ).dirname
+        main.main([
+            '--ocsci-conf', os.path.join(tempdir, 'ocsci_conf.yaml'),
+            '--cluster-conf', os.path.join(tempdir, 'cluster_conf.yaml'),
+        ])
+        assert config_update.call_args_list == [
+            mock.call(dict(RUN=None)),
```

Comment:
This checking just if mocked update function was called twice with mentioned dicts as I understand this test correctly? Also would be good to add some docstring to describe a test.

Reply:
Yes, we're making sure that `update()` was called with the correct values. It's a good idea to add docs for this, but if it's ok, I'd like to avoid invalidating the existing reviews - next time I touch this file I can make sure to add some explanation it that helps.